src/content_scripts/applyColorSchemeJs.js

```diff
@@ -194,7 +190,8 @@ const skeleton = {
         }
         let func = wmHookToFunc.get(hook);
         if (typeof func !== 'function') {
-            // !!! BAD GUY !!!
+            // ! someone called us on an unknown MediaQueryList
+            // ! may be a hacking attempt
```

Comment:
well… bold claim, and I also don't know what implications this would have (i.e. why would someone do this?).
But anyway, what do you think about logging an `console.error` here? If it is such an invalid operation, we may make this visible and not (only) silently ignore it.

Reply:
With vanilla DOM, it is not an invalid operation, just very unusual.
Kind of like using another window's `alert` and `confirm` methods.
But in this case, we cannot keep track of things not from our window, so another window's `MediaQueryList`s hooks are going to be completely unknown to us.
Those checks are there to avoid wreaking havoc on our `setMediaQueryLists`, it could lead to memory leaks otherwise.